client/apis/volumesnapshot/v1/types.go

```diff
@@ -379,7 +379,7 @@ type VolumeSnapshotContentStatus struct {
 	// On Unix, the command `date +%s%N` returns the current time in nanoseconds
 	// since 1970-01-01 00:00:00 UTC.
 	// +optional
-	CreationTime *int64 `json:"creationTime,omitempty" protobuf:"varint,2,opt,name=creationTime"`
+	CreationTime *metav1.Time `json:"creationTime,omitempty" protobuf:"varint,2,opt,name=creationTime"`
```

Comment:
It seems from the comment that this is meant to return the time in nanoseconds, rather than in the time format. I'm also not sure whether you can change the type of a released API...